styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,87 @@
 .footer{
-  
-}
\ No newline at end of file
+  ul{
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+  .footer-menu {
+    li {
+      list-style: none;
+      padding: 0;
+      margin: 8px 0;
+      a {
+        color: darken($light-gray, 15%);
+        font-size: 1.2rem;
+      }
+    }
+    &.split {
+      column-count: 3;
+    }
+  }
+  .footer-accordion-title {
+    color: $white;
+    font-size: 1.3rem;
+  }
+  .footer-tabs {
+    li{
+      a{
+        color:$white;
+      }
+    }
+  }
+}
+@media only screen and (max-width: $screen-sm) {
+  .fluid-footer-container{
+    padding: 0 0;
+  }
+  .footer{
+    padding:0 0;
+    .footer-menu {
+      text-align: center;
+      &.split {
+        column-count: 1;
+      }
+      a{
+        display: block;
+        background-color: lighten($blue-gray, 05%);
+        padding:15px;
+        margin: 0 0;
+        &:hover{
+          background-color: lighten($blue-gray,08%);
+        }
+      }
+      li{
+        margin: 0 0;
+      }
+    }
+    .panel-body{
```

Comment:
Since all the .panel-* are mandated by tab collapse, this should remain inside .footer, but I think anything else, should be in the top-level.